services:

 flaresolverr:

    command:
      - "/usr/local/bin/python"
      - "-u"
      - "/app/flaresolverr.py"

    container_name: "flaresolverr"

   # dns: "10.20.30.55"

    entrypoint:
      - "/usr/bin/dumb-init"
      - "--"

    environment:
      - "LOG_LEVEL=info"
      - "PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
      - "LANG=C.UTF-8"
      - "GPG_KEY=A035C8C19219BA821ECEA86B64E628F8D684696D"
      - "PYTHON_VERSION=3.11.5"
      - "PYTHON_PIP_VERSION=23.2.1"
      - "PYTHON_SETUPTOOLS_VERSION=65.5.1"
      - "PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/9af82b715db434abb94a0a6f3569f43e72157346/public/get-pip.py"
      - "PYTHON_GET_PIP_SHA256=45a2bb8bf2bb5eff16fdd00faef6f29731831c7c59bd9fc2bf1f3bed511ff1fe"

   # hostname: "47d0a4f32647"

    image: "ghcr.io/flaresolverr/flaresolverr:latest"

    ipc: "private"


    logging:
      driver: "json-file"
      options: {}

    network_mode: "container:gluetun"

   # ports:
    #  - "8191:8191/tcp"

    restart: "unless-stopped"

    user: "flaresolverr"

    working_dir: "/app"

    sysctls:
      - net.ipv6.conf.all.disable_ipv6=1

    healthcheck: 
     test: curl --fail http://localhost:8191 || exit 1 
     interval: 10s 
     retries: 5 
     start_period: 5s 
     timeout: 10s